chatbot_production/new.py

```
@@ -0,0 +1,76 @@
+
+pickle.dump(words,open('words.pkl','wb'))
+pickle.dump(classes,open('classes.pkl','wb'))
+
+training=[]
+outputEmpty=[0] * len(classes)
+
+for document in documents:
+    bag=[]
+    wordPatterns=document[0]
+    wordPatterns=[lemmatizer.lemmatize(word.lower()) for word in wordPatterns]
+    for word in words: bag.append(1) if word in wordPatterns else bag.append(0)
```

Ruff (0.13.1)

Multiple statements on one line (colon)

(E701)